flow/probes/targets/netflow_v5.go

```diff
@@ -0,0 +1,260 @@
+		// TCP/UDP destination port number or equivalent
+		binary.Write(records, binary.BigEndian, uint16(dstPort))
+
+		// Unused (zero) bytes
+		binary.Write(records, binary.BigEndian, uint8(0))
+
+		// Cumulative OR of TCP flags
+		binary.Write(records, binary.BigEndian, uint8(0))
+
+		// IP protocol type (for example, TCP = 6; UDP = 17)
+		if f.Network.Protocol == flow.FlowProtocol_TCP {
+			binary.Write(records, binary.BigEndian, uint8(6))
+		} else {
+			binary.Write(records, binary.BigEndian, uint8(17))
+		}
+
+		// IP type of service (ToS)
+		binary.Write(records, binary.BigEndian, uint8(0))
+
+		// Autonomous system number of the source, either origin or peer
+		binary.Write(records, binary.BigEndian, uint16(0))
+
+		// Autonomous system number of the destination, either origin or peer
+		binary.Write(records, binary.BigEndian, uint16(0))
+
+		// Source address prefix mask bits
+		binary.Write(records, binary.BigEndian, uint8(math.MaxUint8))
+
+		// Destination address prefix mask bits
+		binary.Write(records, binary.BigEndian, uint8(math.MaxUint8))
+
+		// Unused (zero) bytes
+		binary.Write(records, binary.BigEndian, uint16(0))
+
+		nFlows++
+	}
+
+	header := new(bytes.Buffer)
+
+	// Version
+	binary.Write(header, binary.BigEndian, uint16(5))
+
+	// Count
+	binary.Write(header, binary.BigEndian, nFlows)
+
+	// SysUpTimeMSecs
+	ms := now.Sub(nf.sysBoot)
+	binary.Write(header, binary.BigEndian, uint32(ms/time.Millisecond))
+
+	// UNIXSecs
+	binary.Write(header, binary.BigEndian, uint32(now.Unix()))
+
+	// UNIXNSecs
+	binary.Write(header, binary.BigEndian, uint32(now.Nanosecond()))
+
+	// SeqNum
+	nf.seqNum += uint32(nFlows)
+	binary.Write(header, binary.BigEndian, nf.seqNum)
+
+	// EngType
+	binary.Write(header, binary.BigEndian, uint8(0))
+
+	// EngID
+	binary.Write(header, binary.BigEndian, uint8(1))
+
+	// SmpInt
+	binary.Write(header, binary.BigEndian, uint16(0))
+
+	nf.conn.Write(append(header.Bytes(), records.Bytes()...))
+}
+
+// SendFlows implements the flow Sender interface
+func (nf *NetFlowV5Target) SendFlows(flows []*flow.Flow) {
+	now := time.Now()
+
+	const bulkSize int = 20
+
+	for i := 0; i < len(flows); i += bulkSize {
+		e := i + bulkSize
+
+		if e > len(flows) {
+			e = len(flows)
+		}
+
+		nf.sendFlowBulk(flows[i:e], now)
+	}
+}
+
+// Start start the target
+func (nf *NetFlowV5Target) Start() {
+	conn, err := net.Dial(nf.URL.Scheme, nf.URL.Host)
+	if err != nil {
+		logging.GetLogger().Errorf("NetFlow socket error: %s", err)
+		return
+	}
+
+	nf.Lock()
+	defer nf.Unlock()
+
+	nf.conn = conn
+	nf.table.Start()
+}
+
+// Stop stops the target
+func (nf *NetFlowV5Target) Stop() {
+	nf.RLock()
+	defer nf.RUnlock()
+
+	nf.table.Stop()
+
+	if nf.conn != nil {
+		nf.conn.Close()
+	}
+}
+
+// NewNetFlowV5Target returns a new NetFlow v5 target
+func NewNetFlowV5Target(capture *types.Capture, nodeTID string) (*NetFlowV5Target, error) {
+	u, err := url.Parse("udp://127.0.0.1:6666")
```

Comment:
according to standard there are several acceptable port numbers - please make url configurable

Reply:
sure, this patch is still in draft mode, will put it in the config in a next iteration